BookingsApi/BookingsApi/Controllers/WorkAllocationsController.cs

```diff
@@ -146,6 +152,13 @@ public async Task<IActionResult> AllocateHearingManually([FromBody] UpdateHearin
                 var list = await _hearingAllocationService.AllocateHearingsToCso(postRequest.Hearings, postRequest.CsoId);
                 
                 var dtos = _hearingAllocationService.CheckForAllocationClashes(list);
+                
+                // need to broadcast acknowledgment message for the allocation
+                if (list.Count > 0)
+                {
+                    await _eventPublisher.PublishAsync(new AllocationHearingsIntegrationEvent(list, list[0].AllocatedTo));
```

Comment:
Should you filter here to only publish hearings for today?

Reply:
it is filtering in a different place